src/index.tsx

```diff
@@ -45,8 +47,20 @@ export function init<Services>(
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const ComponentDataLoader = resources.registerResource<any, LoadArguments<Services>>(
         'component-data-loader',
-        params => {
-            return params.dataDefinition.loadData(params.dataDefinitionArgs, params)
+        ({
+            dataDefinitionArgs,
+            componentRenderPath,
+            dataDefinition,
+            layout,
+            resourceType,
+            ...services
+        }) => {
+            const loadFn = wrapLoad ? wrapLoad(dataDefinition.loadData) : dataDefinition.loadData
```

Comment:
This is where it's done, you could add logging or for us we put the RouteCache hook in here so the SSR can capture the loaded values and put into the route cache.